File: tasks/prac4_python/task10.py
purchase_amount = float(input("Введите сумму покупки: "))
is_regular = input("Вы постоянный клиент? (да/нет): ").lower()

# Базовая скидка
if purchase_amount >= 10000:
    discount = 15
elif purchase_amount >= 5000:
    discount = 10
elif purchase_amount >= 1000:
    discount = 5
else:
    discount = 0

# Дополнительная скидка для постоянных клиентов
if is_regular == "да":
    discount += 3

# Расчет итоговой суммы
if discount > 0:
    final_amount = purchase_amount * (1 - discount / 100)
    print(f"Скидка: {discount}%")
    print(f"Итоговая сумма: {final_amount:.2f}")
else:
    print("Скидка не предоставляется")
    print(f"Итоговая сумма: {purchase_amount:.2f}")

File: tasks/prac4_js/task10.js
const prompt = require('prompt-sync')();

const purchaseAmount = parseFloat(prompt("Введите сумму покупки: "));
const isRegular = prompt("Вы постоянный клиент? (да/нет): ").toLowerCase();

let discount = 0;

// Базовая скидка
if (purchaseAmount >= 10000) {
    discount = 15;
} else if (purchaseAmount >= 5000) {
    discount = 10;
} else if (purchaseAmount >= 1000) {
    discount = 5;
}

// Дополнительная скидка для постоянных клиентов
if (isRegular === "да") {
    discount += 3;
}

// Расчет итоговой суммы
if (discount > 0) {
    const finalAmount = purchaseAmount * (1 - discount / 100);
    console.log(`Скидка: ${discount}%`);
    console.log(`Итоговая сумма: ${finalAmount.toFixed(2)}`);
} else {
    console.log("Скидка не предоставляется");
    console.log(`Итоговая сумма: ${purchaseAmount.toFixed(2)}`);
}